custom/templates/DefaultRevamp/user/placeholders.tpl

```diff
@@ -0,0 +1,102 @@
+{include file='header.tpl'}
+{include file='navbar.tpl'}
+
+<h2 class="ui header">
+  {$TITLE}
+</h2>
+
+{if isset($SUCCESS)}
+  <div class="ui success icon message">
+    <i class="check icon"></i>
+    <div class="content">
+      <div class="header">{$SUCCESS_TITLE}</div>
+      {$SUCCESS}
+    </div>
+  </div>
+{/if}
+
+{if isset($ERRORS)}
+<div class="ui error icon message">
+  <i class="x icon"></i>
+  <div class="content">
+    <ul class="list">
+      {foreach from=$ERRORS item=error}
+      <li>{$error}</li>
+      {/foreach}
+    </ul>
+  </div>
+</div>
+{/if}
+
+<div class="ui stackable grid" id="alerts">
+  <div class="ui centered row">
+    <div class="ui six wide tablet four wide computer column">
+      {include file='user/navigation.tpl'}
+    </div>
+    <div class="ui ten wide tablet twelve wide computer column">
+      <div class="ui segment">
+        <h3 class="ui header">
+          {$PLACEHOLDERS}
+        </h3>
+        <div class="ui middle aligned">
+          {nocache}
+            {if count($PLACEHOLDERS_LIST)}
+            <table class="ui fixed single line selectable unstackable small padded res table">
+              <thead>
+                <tr>
+                  <th>{$SERVER_ID}</th>
+                  <th>{$NAME}</th>
+                  <th>{$VALUE}</th>
+                  <th>{$LAST_UPDATED}</th>
+                  <th>{$SHOW_ON_PROFILE}</th>
+                  <th>{$SHOW_ON_FORUM}</th>
+                </tr>
+              </thead>
+              <tbody>
+              
+              {foreach from=$PLACEHOLDERS_LIST item=data}
+              <tr>
+                <td>
+                  {$data.server_id}
```

Comment:
Is server_id a human friendly variable to show to the end user?

Also, do these variables need purifying?

Reply:
Removed server ID column + purified output of row data